argocd/overlays/moc-infra/applications/envs/moc/smaug/apicurio/kustomization.yaml

```diff
@@ -2,3 +2,4 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
   - registry.yml
+  - api-designer.yml
```

Comment:
file name is different, `.yaml` -> `.yaml` 
hence the ci error, you can also see the error if you try to run `kustomize build .` in this directory